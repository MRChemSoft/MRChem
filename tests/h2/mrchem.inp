# vim:syntax=sh:

rel_prec = 1.0e-4                 # Overall relative precision
printlevel = 0

MRA {
  order = 7                       # Polynomial order of the MW basis
  basis_type = Legendre           # Polynomial type (Legendre or Interpolating)
  min_scale = -4                  # Root scale defines world size 2^{-n}
  max_scale = 25                  # Maximum refinement 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
$coords
H   0.0     0.0    -0.7
H   0.0     0.0     0.7
$end
}

WaveFunction {
  method = HF                     # Wave function method (HF or DFT)
}

Properties {
  scf_energy = true               # Compute ground state energy
}

SCF {
  orbital_thrs = 1.0e-3           # Convergence threshold in orbital residual
  property_thrs = 1.0e-6          # Convergence threshold in total energy
  initial_guess = GTO             # Type of initial guess: none, mw, gto
  kain = 3                        # Length of KAIN iterative history
}

