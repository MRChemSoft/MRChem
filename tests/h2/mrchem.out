


************************************************************
***                                                      ***
***    MRChem 1.0.0 (rev. e6faeac)                       ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Jonas Juselius   <jonas.juselius@uit.no>          ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***                                                      ***
************************************************************


Print level  : 0

BLAS was NOT found, Eigen will be used instead!

+++ Parallel execution: 
  MPI hosts available     : 1
  Threads/host            : 16
  Total used CPUs         : 16



============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 7
 polynomial type  = Legendre
------------------------------------------------------------
 unit length      = 16.00000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [   -16.00000   -16.00000   -16.00000 ]
 upper bounds     = [    16.00000    16.00000    16.00000 ]
 total length     = [    32.00000    32.00000    32.00000 ]
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     H            1.00000e+00   1.00000e-04   7.57698e-03
  2     H            1.00000e+00   1.00000e-04   7.57698e-03
============================================================


============================================================
        Setting up occupied orbitals (closed-shell)
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    u    2                       9.999999999846607e-01
------------------------------------------------------------
                 Wall time: 2.64800e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     712      4.76150e-02
 Coulomb density                       328      2.08040e-02
 Coulomb potential                     136      1.91597e-01
 Hartree-Fock exchange                 264      2.14082e-01
------------------------------------------------------------
                 Wall time: 4.74241e-01 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              1.00000e-04
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        4.95000e-04
 Diagonalizing matrix                           5.00000e-06
 Rotating orbitals                              8.80700e-03
------------------------------------------------------------
                 Wall time: 9.35200e-03 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       7.142857142857143e-01
 Electronic energy                   -1.842584621984340e+00
------------------------------------------------------------
 Total energy                        -1.128298907698626e+00
------------------------------------------------------------
                 Wall time: 3.50160e-02 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -5.95939e-01
------------------------------------------------------------
                 Wall time: 1.47652e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.94187e-01  0.00000e+00   328     0  5.8e-03  6.1e-01
------------------------------------------------------------
                 Wall time: 6.06786e-01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             7.00000e-06
 Copy orbitals                                  3.07100e-03
 Copy orbital updates                           5.08100e-03
------------------------------------------------------------
                 Wall time: 8.24500e-03 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     712      4.68750e-02
 Coulomb density                       392      2.10410e-02
 Coulomb potential                     136      1.97175e-01
 Hartree-Fock exchange                 392      2.31251e-01
------------------------------------------------------------
                 Wall time: 4.96509e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.96461e-01   1.82519e-02        392    u    2    0
------------------------------------------------------------
 Total error:                            1.82519e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       1.073477856683062e+00     1.07348e+00    0
 N-E          -3.566768512638604e+00    -3.56677e+00    0
 Coulomb       1.301412067963103e+00     1.30141e+00    0
 Exchange     -6.507060339919013e-01    -6.50706e-01    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -1.842584621984340e+00    -1.84258e+00    0
 Nuclear       7.142857142857143e-01     7.14286e-01    0
------------------------------------------------------------
 Total        -1.128298907698626e+00    -1.12830e+00    0
============================================================


################ Wall time: 1.41608e+00 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              1.00000e-04
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        1.00700e-03
 Diagonalizing matrix                           4.00000e-06
 Rotating orbitals                              8.39300e-03
------------------------------------------------------------
                 Wall time: 9.44700e-03 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       7.142857142857143e-01
 Electronic energy                   -1.847569710412104e+00
------------------------------------------------------------
 Total energy                        -1.133283996126390e+00
------------------------------------------------------------
                 Wall time: 4.02470e-02 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -5.96461e-01
------------------------------------------------------------
                 Wall time: 1.49973e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.96916e-01  0.00000e+00   328     0  3.1e-03  6.3e-01
------------------------------------------------------------
                 Wall time: 6.31564e-01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             3.00000e-06
 Copy orbitals                                  3.72500e-03
 Copy orbital updates                           4.43100e-03
------------------------------------------------------------
                 Wall time: 8.23800e-03 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     712      4.71360e-02
 Coulomb density                       392      2.08090e-02
 Coulomb potential                     136      2.01593e-01
 Hartree-Fock exchange                 392      2.33714e-01
------------------------------------------------------------
                 Wall time: 5.03417e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.95366e-01   5.13547e-03        392    u    2    0
------------------------------------------------------------
 Total error:                            5.13547e-03    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       1.104177300303831e+00     3.06994e-02    0
 N-E          -3.606394276126355e+00    -3.96258e-02    0
 Coulomb       1.309294530820841e+00     7.88246e-03    0
 Exchange     -6.546472654104201e-01    -3.94123e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -1.847569710412104e+00    -4.98509e-03    0
 Nuclear       7.142857142857143e-01     0.00000e+00    1
------------------------------------------------------------
 Total        -1.133283996126390e+00    -4.98509e-03    0
============================================================


################ Wall time: 1.45292e+00 sec ################




####################### SCF cycle  3 #######################


============================================================
 Current precision                              1.00000e-04
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       7.142857142857143e-01
 Electronic energy                   -1.847891541168436e+00
------------------------------------------------------------
 Total energy                        -1.133605826882722e+00
------------------------------------------------------------
                 Wall time: 4.08090e-02 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -5.95366e-01
------------------------------------------------------------
                 Wall time: 1.56041e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.98924e-01  0.00000e+00   328     0  1.1e-03  6.3e-01
------------------------------------------------------------
                 Wall time: 6.33110e-01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             3.00000e-06
 Setup linear system                            1.14780e-02
 Solve linear system                            1.20000e-05
 Expand solution                                1.23710e-02
------------------------------------------------------------
                 Wall time: 2.39490e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     712      4.69520e-02
 Coulomb density                       392      2.07080e-02
 Coulomb potential                     136      2.02431e-01
 Hartree-Fock exchange                 392      2.36378e-01
------------------------------------------------------------
                 Wall time: 5.06640e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.94695e-01   2.36509e-03        392    u    2    0
------------------------------------------------------------
 Total error:                            2.36509e-03    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       1.118661740831658e+00     1.44844e-02    0
 N-E          -3.623711872824860e+00    -1.73176e-02    0
 Coulomb       1.314317181649532e+00     5.02265e-03    0
 Exchange     -6.571585908247662e-01    -2.51133e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -1.847891541168436e+00    -3.21831e-04    0
 Nuclear       7.142857142857143e-01     0.00000e+00    1
------------------------------------------------------------
 Total        -1.133605826882722e+00    -3.21831e-04    0
============================================================


################ Wall time: 1.47358e+00 sec ################




####################### SCF cycle  4 #######################


============================================================
 Current precision                              1.00000e-04
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       7.142857142857143e-01
 Electronic energy                   -1.847922940522545e+00
------------------------------------------------------------
 Total energy                        -1.133637226236830e+00
------------------------------------------------------------
                 Wall time: 4.16820e-02 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -5.94695e-01
------------------------------------------------------------
                 Wall time: 1.73994e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.99977e-01  0.00000e+00   328     0  2.3e-05  6.4e-01
------------------------------------------------------------
                 Wall time: 6.44523e-01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.86300e-03
 Setup linear system                            3.41070e-02
 Solve linear system                            1.50000e-05
 Expand solution                                1.90920e-02
------------------------------------------------------------
                 Wall time: 5.51770e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     712      4.69590e-02
 Coulomb density                       392      2.12720e-02
 Coulomb potential                     136      2.05563e-01
 Hartree-Fock exchange                 392      2.34229e-01
------------------------------------------------------------
                 Wall time: 5.08197e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.94671e-01   1.25540e-04        392    u    2    1
------------------------------------------------------------
 Total error:                            1.25540e-04    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       1.125906955842972e+00     7.24522e-03    0
 N-E          -3.632362920979245e+00    -8.65105e-03    0
 Coulomb       1.317066049227456e+00     2.74887e-03    0
 Exchange     -6.585330246137279e-01    -1.37443e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -1.847922940522545e+00    -3.13994e-05    0
 Nuclear       7.142857142857143e-01     0.00000e+00    1
------------------------------------------------------------
 Total        -1.133637226236830e+00    -3.13994e-05    0
============================================================


################ Wall time: 1.53664e+00 sec ################




####################### SCF cycle  5 #######################


============================================================
 Current precision                              1.00000e-04
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       7.142857142857143e-01
 Electronic energy                   -1.847922949940644e+00
------------------------------------------------------------
 Total energy                        -1.133637235654930e+00
------------------------------------------------------------
                 Wall time: 4.13850e-02 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Re-using operator with lambda                 -5.94695e-01
------------------------------------------------------------
                 Wall time: 5.00000e-06 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.99995e-01  0.00000e+00   328     0  4.9e-06  6.3e-01
------------------------------------------------------------
                 Wall time: 6.32878e-01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             2.32700e-03
 Setup linear system                            3.30900e-02
 Solve linear system                            1.00000e-05
 Expand solution                                1.91780e-02
------------------------------------------------------------
                 Wall time: 5.47140e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     712      4.67690e-02
 Coulomb density                       392      2.08450e-02
 Coulomb potential                     136      2.00381e-01
 Hartree-Fock exchange                 392      2.47106e-01
------------------------------------------------------------
                 Wall time: 5.15263e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -5.94668e-01   1.32135e-05        392    u    2    1
------------------------------------------------------------
 Total error:                            1.32135e-05    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       1.126020978131420e+00     1.14022e-04    0
 N-E          -3.632524668092044e+00    -1.61747e-04    0
 Coulomb       1.317161480039959e+00     9.54308e-05    0
 Exchange     -6.585807400199793e-01    -4.77154e-05    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -1.847922949940644e+00    -9.41810e-09    1
 Nuclear       7.142857142857143e-01     0.00000e+00    1
------------------------------------------------------------
 Total        -1.133637235654930e+00    -9.41810e-09    1
============================================================


################ Wall time: 1.35726e+00 sec ################


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        1.17600e-03
 Diagonalizing matrix                           4.00000e-06
 Rotating orbitals                              4.68200e-03
------------------------------------------------------------
                 Wall time: 5.90900e-03 sec 
============================================================


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    1.82519e-02    -1.128298907698626e+00   -1.12830e+00
  2    5.13547e-03    -1.133283996126390e+00   -4.98509e-03
  3    2.36509e-03    -1.133605826882722e+00   -3.21831e-04
  4    1.25540e-04    -1.133637226236830e+00   -3.13994e-05
  5    1.32135e-05    -1.133637235654930e+00   -9.41810e-09
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                     712      4.68190e-02
 Coulomb density                       392      2.09570e-02
 Coulomb potential                     136      2.04194e-01
 Hartree-Fock exchange                 392      2.55304e-01
------------------------------------------------------------
                 Wall time: 5.27425e-01 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 4.05590e-02 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-5.94668e-01
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    u    2                      -5.946676708306496e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     H           0.00000e+00   0.00000e+00  -7.00000e-01
  2     H           0.00000e+00   0.00000e+00   7.00000e-01
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -1.189335341661299e+00
 Kinetic energy:                      1.126050027436847e+00
 E-N energy:                         -3.632560603202188e+00
 Coulomb energy:                      1.317175234104042e+00
 Exchange energy:                    -6.585876170520206e-01
 X-C energy:                          0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Electronic energy                   -1.847922958713321e+00
 Nuclear energy                       7.142857142857143e-01
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -1.133637244427606e+00
                    (kJ/mol)         -2.976364154462528e+03
                    (kcal/mol)       -7.113681052893904e+02
                    (eV)             -3.084783956514056e+01
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 8.724940e+00                ***
***                                                      ***
************************************************************


